react-api-client/Makefile

```
@@ -20,7 +20,7 @@ all: clean build
 
 .PHONY: clean
 clean:
-	yarn shx rm -rf dist
+	pnpm exec shx rm -rf \"dist\"
```

I think we actually need the original, without quotes:

```suggestion
	pnpm exec shx rm -rf dist
```

`\"dist\"` tries to delete a directory whose name is literally `./"dist"`, including the quotes, instead of `./dist`.